dev-support/test-patch.sh

```diff
@@ -2963,15 +2572,14 @@ function check_unittests
   local needlog
   local unitlogs
 
-  big_console_header "Running unit tests"
-
   verify_needed_test unit
 
   if [[ $? == 0 ]]; then
-    echo "Existing unit tests do not test patched files. Skipping."
     return 0
   fi
 
+  big_console_header "Running unit tests"
```

Comment:
shellcheck: 5: note: Double quote to prevent globbing and word splitting. [SC2086]